### PR TITLE
Fix offline benchmark

### PR DESCRIPTION
**Pull request by cwpeng-cn**
fix offline-benchmark and add BenchmarkGuide.md

### CODE REVIEW

File: zoo/src/test/resources/serving/BenchmarkGuide.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+## offline benchmark guide
```
Comment (qiyuangong) [inline zoo/src/test/resources/serving/BenchmarkGuide.md:1]: Offline

File: zoo/src/test/resources/serving/cluster-serving-enqueue-test.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+img = cv2.imread(path)
+img = cv2.resize(img, (224, 224))
+data = cv2.imencode(".jpg", img)[1]
+img_encoded = base64.b64encode(data).decode("utf-8")
+input_api.enqueue("my-img", t={"b64": img_encoded})
+time.sleep(5)
+
+for i in range(10000):
+    input_api.enqueue("my-img", t={"b64": img_encoded})
+print("10000 images enqueued")
```
Comment (qiyuangong) [inline zoo/src/test/resources/serving/cluster-serving-enqueue-test.py:24]: Empty line at the end

File: zoo/src/test/resources/serving/BenchmarkGuide.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+##### STEP 2:
+```
+cd analytics-zoo/zoo/src/test/resources/serving
+```
+
+##### STEP 3 (Optional) :
+```
+cluster-serving-init
+```
+The script will generate `config.yaml` and `zoo.jar` in the current path.
```
Comment (qiyuangong) [inline zoo/src/test/resources/serving/BenchmarkGuide.md:20]: This maybe better

File: zoo/src/test/resources/serving/BenchmarkGuide.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+If these files already exist, skip STEP 3.
+
+
+##### STEP 4:
+Put the model in any of your local directory, and set model:/path/to/dir in `config.yaml`.
```
Comment (qiyuangong) [inline zoo/src/test/resources/serving/BenchmarkGuide.md:26]: remove any of

File: zoo/src/test/resources/serving/cluster-serving-enqueue-test.py (hunk 2 above)
Comment (cwpeng-cn) [reply zoo/src/test/resources/serving/cluster-serving-enqueue-test.py:24]: Thanks

**Pull request closed without merging**